src/main/java/ru/yandex/practicum/filmorate/storage/dao/film/DirectorDao.java

```diff
@@ -0,0 +1,27 @@
+package ru.yandex.practicum.filmorate.storage.dao.film;
+
+import ru.yandex.practicum.filmorate.model.Director;
+import ru.yandex.practicum.filmorate.model.Film;
+
+import java.util.List;
+
+public interface DirectorDao {
+
+    Director createDirector(Director director);
+
+    Director updateDirector(Director director);
+
+    List<Director> findAllDirectors();
+
+    Director findDirectorById(Integer id);
```

Comment:
Аналогично выше сказанному.
```suggestion
    Director getById(Integer id);
```